static/faq.html

```diff
@@ -361,6 +367,23 @@ <h3 id="cellular-tracking">
             sending texts or other data is not required or particularly useful to track devices
             connected to a network for an adversary with the appropriate access.</p>
 
+            <h2 id="day-to-day-use">
+                <a href="#day-to-day-use">Day to day use</a>
+            </h2>
+
+            <h3 id="vpn-support">
+                <a href="#vpn-support">Does GrapheneOS support VPNs and VPN apps?</a>
+            </h3>
+
+            <p>Yes. Orbot, OpenVPN for Android, the userspace implementation of WireGuard, and the
+            Private Internet Access VPN app have all been tested and reported working. VPNs can be
+            configured under Settings -> Network & Internet -> Advanced -> VPN.</p>
+
+            <p>GrapheneOS has built-in VPN support included in the operating system and has an
```

Comment:
Built-in VPN support should be the first paragraph, followed by app support, and then always-on VPN support. Always-on VPN support should be a separate paragraph. As is, this is going to be confusing, since it makes it seem like always-on VPN access is only for built-in VPN support.

Reply:
Okay! I've tried reordering it for this one, looks like it made it through after a rebase.